website/project/model.py

```diff
@@ -823,8 +817,7 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     contributors = fields.ForeignField('user', list=True)
     users_watching_node = fields.ForeignField('user', list=True)
 
-    logs = fields.ForeignField('nodelog', list=True, backref='logged')
-    tags = fields.ForeignField('tag', list=True)
+    tags = fields.ForeignField('tag', list=True, backref='tagged')
```

Comment:
I'm not sure if the diff is being weird or something, but the `tagged` backref was removed recently. Could you go ahead and remove it from this PR?